test/test_integration.py

```diff
@@ -54,58 +56,129 @@ def tearDown(self):
-        body = Subscription(account_id='4eba7df9-f023-4425-bb46-b1a2e6e84fea',
-                            plan_name='sports-monthly',
-                            product_name='Basic',
-                            billing_period='MONTHLY',
-                            price_list='DEFAULT')
-        subscription.create_subscription(body, 'test', 'bob', 'lazar')
+        body = Subscription(account_id=account_id, plan_name='standard-monthly')
+        subscription.create_subscription(body, 'test', random_api_key, random_api_secret)
+
+        # Get account bundles
+        account = killbill.api.AccountApi()
+        bundles = account.get_account_bundles(account_id, random_api_key, random_api_secret)
+        subscription_id = bundles[0].subscriptions[0].subscription_id
+
+        # Get subscription
+        subscription = killbill.api.SubscriptionApi()
+        subscription = subscription.get_subscription(subscription_id, random_api_key, random_api_secret)
+        self.assertEqual('standard-monthly', subscription.plan_name)
+
+        # Get account invoices
+        account = killbill.api.AccountApi()
+        invoices = account.get_invoices_for_account(account_id, random_api_key, random_api_secret)
+        self.assertEqual([], invoices)
+
+        # Remove AUTO_INVOICING_OFF tag
+        account = killbill.api.AccountApi()
+        account.delete_account_tags(account_id, 'test', random_api_key, random_api_secret, tag_def=tag)
+
+        # Get account invoices
+        account = killbill.api.AccountApi()
+        invoices = account.get_invoices_for_account(account_id, random_api_key, random_api_secret)
+        self.assertIsNotNone(invoices[0].invoice_id)
+
+        # Create a dryRun invoice
+        invoice = killbill.api.InvoiceApi()
+        body = InvoiceDryRun(dry_run_type='TARGET_DATE')
+        invoice.generate_dry_run_invoice(body, account_id, 'test', random_api_key, random_api_secret)
+
+        # Modify Plan
+        subscription = killbill.api.SubscriptionApi()
+        body = Subscription(subscription_id=subscription_id, product_name='Super', billing_period='MONTHLY', price_list='DEFAULT')
+        subscription.change_subscription_plan(subscription_id, body, 'test', random_api_key, random_api_secret, billing_policy='IMMEDIATE')
+
+        # Get account invoice payments
+        account = killbill.api.AccountApi()
+        invoice_payments = account.get_invoice_payments(account_id, random_api_key, random_api_secret)
+
+        # Cancel subscription
+        subscription = killbill.api.SubscriptionApi()
+        subscription.cancel_subscription_plan(subscription_id, 'test', random_api_key, random_api_secret)
+
+        # Get subscription
+        subscription = killbill.api.SubscriptionApi()
+        subscription = subscription.get_subscription(subscription_id, random_api_key, random_api_secret)
+        self.assertEqual('CANCELLED', subscription.state)
+
+
+        # Pagination: List Accounts
```

Comment:
It would be better to create a second scenario for the pagination/search use case and create more accounts/payment methods. Also add some assertion to make sure those work as expected